core/src/test/scala/pink/cozydev/lucille/ParserSuite.scala

```diff
@@ -22,8 +22,10 @@ import Parser._
 
 class SingleSimpleQuerySuite extends munit.FunSuite {
 
-  def assertSingleTerm(r: Either[Error, NonEmptyList[Query]], expected: Query) =
-    assertEquals(r, Right(NonEmptyList.of(expected)))
+  def assertSingleTerm(r: Either[Error, MultiQuery], expected: Query)(implicit
+      loc: munit.Location
+  ) =
+    assertEquals(r, Right(MultiQuery(NonEmptyList.one(expected))))
```

Comment:
Do you still need the non empty list here? Can't you just pass `expected` as the `head` to MultiQuery?

Reply:
Nice catch